src/features/foods/types/FoodTypes.ts

```diff
@@ -0,0 +1,7 @@
+import { IContainer, IFood } from '@/types/definition';
+
+export interface IFoodView extends IFood {
+  container: string;
+}
+export type GroupIdContainersMapType = Record<IContainer['groupId'], IContainer['id'][]>;
```

Comment:
I thought `GroupIdContainersMapType` should be in `FoodTypes.ts` as this is referred from more than one file (`FilterButton.tsx` and `containerMappings.tsx`)